bw2io/strategies/generic.py

```diff
@@ -171,14 +406,57 @@ def normalize_units(db):
 
 
 def add_database_name(db, name):
-    """Add database name to datasets"""
+    """
+    Add a database name to the datasets.
+
+    Parameters
+    ----------
+    db : list[dict]
+        The list of datasets to add the database name to.
+    name : str
+        The name of the database to be added.
```

Comment:
"Name __of the database to be added__" implies that this functions adds a database. As far as I see it only adds the name. Best to reformulate this.